### PR TITLE
test-spect: remove mpsl&sdc from some CI labels

### PR DESCRIPTION
**Pull request by guwa**
remove few CI from mpsl&sdc changes

### CODE REVIEW

File: .github/test-spec.yml [hunk 1]
```diff
@@ -37,10 +37,6 @@
   - "mpsl/lib/**/*"
 
 "CI-mesh-test":
-  - "softdevice_controller/include/**/*"
-  - "softdevice_controller/lib/**/*"
-  - "mpsl/include/**/*"
-  - "mpsl/lib/**/*"
```
Comment (PerMac) [inline .github/test-spec.yml:43]: is leaving empty entries accepted?

Comment (thst-nordic) [reply .github/test-spec.yml:43]: according to yml spec empty should get convert to null safely.  
but you could add it to be safe. 
`"CI-mesh-test": null`

**Review by miha-nordic — APPROVED** [review]

**Review by doublemis1 — APPROVED** [review]
LGTM

**Review by mariusz-nordicsemi — APPROVED** [review]
Matter tests disabled as changes are verified by lower layers (Radio Software and Thread tests) and BLE itself. Matter verification would be performed in nightly plan.
In case of any regression the idea is to increase test coverage in BLE plan. And if that wouldn't work, the Matter plan could be enabled again.

**Pull request closed without merging**